sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -258,6 +264,33 @@ public partial class DataLakeUriBuilder
 }
 namespace Azure.Storage.Files.DataLake.Models
 {
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct AccessControlRecursiveChangeFailure
```

Comment:
`Change` is a noun here. I wasn't 100% sure if I should leave or drop `Recursive` part - mostly because we have non-recursive APIs that touch ACL, so trying to avoid confusion.

Reply:
I don't think anyone's going to be looking directly at these `.Models` types when they're getting started.  Almost all of our customers begin with the client and figure out the options/results once they've narrowed things down to the right method.